src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/TimerManager.cs

```diff
@@ -113,6 +135,8 @@ private void DisableWindowsUpdate()
                         auUtility.SetAUOptions();
                         _eventSource.InfoMessage("New AU registry values are {0}", auUtility.LogCurrentAUValues());
                     }
+                    string updateMsg = "Automatic Windows update disabled successfully.";
```

Comment:
>Automatic Windows update disabled successfully. [](start = 40, length = 47)

Change the message to, Windows Update has been configured to "Notify before Download".